docs/root/extending/extending.rst

```diff
@@ -19,8 +19,11 @@ types including:
 * :ref:`Stat sinks <arch_overview_statistics>`
 * :ref:`Tracers <arch_overview_tracing>`
 * :ref:`Request ID <arch_overview_tracing>`
-* Transport sockets
-* BoringSSL private key methods
+* :ref:`Transport sockets <envoy_v3_api_field_extensions.transport_sockets.tls.v3.CommonTlsContext.CertificateProvider.typed_config>`
+* :ref:`BoringSSL FIPS <arch_overview_ssl_fips>`
```

Comment:
Is this an extension point? Did you mean to ref link to the private key methods or have a different title?

Reply:
will revert the BoringSSL FIPs since I'm not entirely sure. I was basing this on pr #6326 but might be missing some context.